### ISSUE TITLE
Making a vector from a list of scalars: Add stack()?

### ISSUE THREAD

**Issue by amilsted** [google/TensorNetwork#350]
I want to take a list of scalar values and make a vector of it using backend operations. This seems to be tricky at the moment! My initial attempt was:
```python
scalars = [backend.convert_to_tensor(1.0), backend.convert_to_tensor(2.0)]
vec = backend.concat(scalars, axis=0)
```
This fails, since I cannot concatenate zero-dimensional arrays.

Perhaps we should have a backend.stack()?

**Comment by Thenerdstation** [comment]
Yes, backend.concat is ONLY for shape calculations (we really should change the name).

I'd prefer to change backend.concat to backend.shape_concat.
